config/owasp/suppressions.xml

```diff
@@ -42,7 +42,7 @@
   </suppress>
 
   <suppress>
-    <notes>This is transitive dependency coming for spring boot starter, CVE is for tomcat-embed-core-9.0.39.jar and tomcat-embed-websocket-9.0.39.jar</notes>
+    <notes>This is transitive dependency coming from spring boot starter, CVE is for tomcat-embed-core/Websocket jar. Latest version also have same CVE issues so expiry date cant be mentioned</notes>
```

Comment:
you should still put an expiry date, ~2 weeks should give time for it to be fixed

Reply:
@timja It's no longer applicable as CVE suppression is removed now